pubsub/consumer.go

```diff
@@ -36,6 +36,8 @@ var TestConsumerConfig = ConsumerConfig{
 	IdletimeToAutoclaim:  30 * time.Millisecond,
 }
 
+var AlreadySetError = errors.New("redis key already set")
```

Comment:
I thought the style (enforced by lint - I thought) was to name thees things like, `ErrAlreadySet`
Or, do I have it backwards?

Reply:
Yes.

<img width="651" alt="image" src="https://github.com/user-attachments/assets/9dc09779-9d89-4ef8-8742-9d4f5ee83111" />


This is what I see in VSCode on this branch.